arviz/utils.py

```diff
@@ -5,6 +5,19 @@
 import re
 import warnings
 from functools import lru_cache
+from copy import deepcopy
+
+from .data import InferenceData, convert_to_dataset, convert_to_inference_data
+from typing import cast
+from .rcparams import rcParams, ScaleKeyword, ICKeyword
+
+NO_GET_ARGS: bool = False
+try:
+    from typing_extensions import get_args
+except ImportError:
+    NO_GET_ARGS = True
+
+from arviz.stats.stats_utils import ELPDData
```

Comment:
This looks like a circular import issue, I think we should move the `calculate_ics` to `stats_utils.py` file to prevent it.